src/content/posts/How-to-test-Angor/index.md

```diff
@@ -17,88 +17,107 @@ Angor is a decentralized crowdfunding project leveraging the security of Bitcoin
 ### Step 1: Initial Setup
 
 #### 1. Create a Wallet on Angor
-**Steps:**
+Steps:
 - Sign up on the Angor platform.
 - Navigate to the wallet creation section.
 - Click on "Create Wallet."
 - Angor will automatically set up the wallet for you.
 
 #### 2. Obtain Test Coins
-**Steps:**
+Steps:
 - Go to the test coins section.
 - Click on "Get Test Coins."
 - The test coins will be added to your newly created wallet.
 
-## Step 2: Testing the Investment Process
+### Step 2: Testing the Investment Process
 
-### 1. Choose a Project
-**Steps:**
+#### 1. Choose a Project
+Steps:
 - Browse the available projects on Angor.
 - Select a project to invest in.
 - Review the project details and milestones.
 
-### 2. Make an Investment
-**Steps:**
+#### 2. Make an Investment
+Steps:
 - Navigate to the chosen project’s page.
 - Click on the "Invest" button.
 - Enter the amount you wish to invest in the provided field.
 - Click "Submit" to confirm the transaction.
 - Wait for the founder to approve (this is a manual process by the founder).
 - Wait for the transaction to be confirmed on the blockchain. This may take a few minutes.
 
-### 3. If you are testing as the founder and investor
-**Steps:**
+#### 3. If you are testing as the founder and investor
 - Go to the project page and approve the investment.
 
-## Step 3: Testing Milestone Verification and Fund Release
+### Step 3: Creating and Managing a Project (For Founders)
 
-### 1. Monitor Project Progress
-**Steps:**
+#### Create a New Project
+Steps:
+- Navigate to the "Create Project" section on Angor.
+- Enter the project name, description, and goals.
+- Define the project milestones and the corresponding time-lock contracts.
```

Comment:
Maybe also mention to add a banner this is important for projects view page

Reply:
Yes will commit that